### PR TITLE
cleanup after cloud.gov exploration

### PR DESCRIPTION
**Pull request by mogul**
Cleans up now that https://github.com/GSA/datagov-deploy/issues/917 has spawned a bunch of issues with this information.

### CODE REVIEW

**Comment by mogul** [comment]
Issue moved to [GSA/datagov-deploy #1159](https://github.com/GSA/datagov-deploy/issues/1159) via [**ZenHub**](https://www.zenhub.com/)

File: manifest.yml [hunk 1]
```diff
@@ -17,4 +17,17 @@ applications:
...
-    APP_DIR: /home/vcap/app/htdocs
\ No newline at end of file
+    APP_DIR: /home/vcap/app/htdocs
+  processes:
+  - type: web
+    command: $HOME/.bp/bin/start
+    disk_quota: 512M
+    memory: 256M
+    timeout: 10
+  - type: worker
```
Comment (adborden) [inline manifest.yml]: Are we not going the CI scheduler route?

**Comment by mogul** [comment]
We are, and [I've documented that here](
https://github.com/GSA/datagov-deploy/issues/1011), but this is the current
implementation that Peter left in the branch before we made that decision.

**Comment by adborden** [comment]
Okay, so you're saying we'll go with this implementation, and then come back to CI later? Is there an issue capturing that or should there be?

**Comment by mogul** [comment]
Yeah, that's this one. I'll change the title to make it clearer that it's about driving those jobs from CI.
https://github.com/GSA/datagov-deploy/issues/1011

**Pull request merged**